### PR TITLE
update with latest common

### PR DESCRIPTION
**Pull request by Fatme**

### CODE REVIEW

**Comment by rosen-vladimirov** [comment]
You should gitignore docs/html dir

**Comment by teobugslayer** [comment]
:+1:

File: docs/man_pages/index.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
+#AppBuilder
```
Comment (rosen-vladimirov) [inline docs/man_pages/index.md]: This name should be changed. Also the file contains non-ddb commands

**Comment by rosen-vladimirov** [comment]
:+1:

**Pull request merged**